.circleci/config.yml

```
@@ -32,3 +32,16 @@ workflows:
             branches:
               only:
                 - master
+      - codacy/shell:
+          name: tag_version_with_v
+          cmd: |
+            VERSION=$(cat .version)
+            echo "v$VERSION" > .version
+          persist_to_workspace: true
+          requires:
+            - unit_test
+      - codacy/tag_version:
+          name: tag_version
+          context: CodacyAWS
+          requires:
+            - tag_version_with_v
```

They could run in parallel
```suggestion
            - unit_test
```